src/translations/en.json

```
@@ -2144,6 +2144,7 @@
             "connection_status": "Cloud connection status",
             "manage_account": "Manage Account",
             "sign_out": "Sign out",
+            "sign_out_confirm": "Are you sure you want to log out from the Home Assistant Cloud?",
```

We can drop the name, context is clear.

The button is called "Sign out" so we should use same verb.

```suggestion
            "sign_out_confirm": "Are you sure you want to sign out?",
```